src/Mutator/Removal/MatchArmRemoval.php

```diff
@@ -116,13 +117,13 @@ public function mutate(Node $node): iterable
         foreach ($node->arms as $i => $arm) {
             $arms = $node->arms;
 
-            if ($arm->conds !== null && count((array) $arm->conds) > 1) {
+            if (count((array) $arm->conds) > 1) {
                 foreach ($arm->conds as $j => $cond) {
                     $conds = $arm->conds;
```

Comment:
unrelated, so best in a different PR, but since the value is used L120 I think this assignment should precede L120, which can then make use of this variable